src/lib/components/viewer/AnnotationLayer.svelte

```diff
@@ -334,44 +350,49 @@ Assumes it's a child of a ViewerContext
   }
 
   .box.public {
-    border-color: var(--note-public);
-    background: var(--note-public);
+    border-color: color-mix(in srgb, var(--yellow-3), var(--gray-4));
+    background: var(--yellow-3);
```

Comment:
I'd like to keep the semantic names for note colors, if only because I could see us changing them later.